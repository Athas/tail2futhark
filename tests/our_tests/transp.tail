let v0:[int]1 = iotaV(8) in
let v1:[int]1 = transp(v0) in
v1
