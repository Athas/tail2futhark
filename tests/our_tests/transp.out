fun real main() =
  let t_v0 = iota(8) in
  let t_v1 = transpose(t_v0) in
  t_v1
